pyanaconda/payload/dnf/payload.py

```diff
@@ -1521,8 +1532,15 @@ def update_base_repo(self, fallback=True, checkmount=True):
         # We need to check this again separately in case REPO_FILES were set above.
         if source_type in SOURCE_REPO_FILE_TYPES:
 
-            # If this is a kickstart install, just return now
-            if flags.automatedInstall:
+            # If this is a kickstart install, just return now as we normally do not
+            # want to read the on media repo files in such a case. On the other hand,
+            # the local repo files are a valid use case if the system is subscribed
+            # and the CDN is selected as the installation source.
+            subscription_proxy = SUBSCRIPTION.get_proxy()
+            cdn_source = self.source_type == SOURCE_TYPE_CDN
```

Comment:
Better name would be `is_cdn_source` here.

Reply:
Changed.